README.rst

```diff
@@ -5,9 +5,9 @@ Weaviate python client
     :align: right
```

Comment:
The image logo location would be changed too (`https://raw.githubusercontent.com/weaviate/weaviate`). Maybe we can add it in this repo?

Reply:
Bob said we will handle that later

Reply:
Isn't https://github.com/weaviate/weaviate-python-client/blob/main/docs/index.rst updated automatically? It should be generated from the readme?

> https://github.com/weaviate/weaviate-python-client/blob/main/.github/CODEOWNERS

Good catch :)

Reply:
> Isn't https://github.com/weaviate/weaviate-python-client/blob/main/docs/index.rst updated automatically? It should be generated from the readme?

No. It is generated from all the `*.rst` files in the `docs/`. It is the home page for the readthedocs.org. So it will not work if we use the README.rst.

I meant to change the URL and the name for this line:
> Visit the official [SeMi Technology](https://www.semi.technology/) website for more information about the Weaviate and how to use it in production.